src/main/kotlin/com/healthmetrix/labres/order/OrderController.kt

```diff
@@ -68,14 +112,24 @@ data class CreateOrderRequestBody(val notificationId: String)
 
 data class UpdateOrderRequestBody(val notificationId: String)
 
-sealed class UpdateOrderResponse(httpStatus: HttpStatus) : ApiResponse(httpStatus, false) {
+sealed class UpdateOrderResponse(httpStatus: HttpStatus) : LabResApiResponse(httpStatus, false) {
     object Updated : UpdateOrderResponse(HttpStatus.OK)
     object NotFound : UpdateOrderResponse(HttpStatus.NOT_FOUND)
 }
 
-sealed class CreateOrderResponse(httpStatus: HttpStatus, hasBody: Boolean = true) : ApiResponse(httpStatus, hasBody) {
+sealed class CreateOrderResponse(httpStatus: HttpStatus, hasBody: Boolean = true) :
+    LabResApiResponse(httpStatus, hasBody) {
+
     data class Created(
+        @Schema(
+            description = "A unique internal identifier for the order",
+            example = "65e524cb-7494-4073-ad16-495fed0d79e4"
+        )
         val id: UUID,
+        @Schema(
+            description = "numeric 10-digit-long identifier",
```

Comment:
EONs are mentioned above, should we reuse that vocab here or remove EON vocab from above?